### ISSUE TITLE
Status check: When new version out?

### ISSUE THREAD

**Issue by elvary** [xbrowsersync/app#424]
It's been over 2 years since last update ( v1.5.2 on may 5th 2020), im using this xbrowsersync extension in hope of one day sync history will be available, im exhausted of the way things are with incompatible google account sync history and brave sync chain, all the csv json sql imports/exports how damn complicated and difficult they are to unite into one central point of storage;
![image](https://user-images.githubusercontent.com/69122770/189770428-9bf6a02b-36e5-4e93-9dbd-1a7993b9e246.png)
 i need xbrowsersync feature sync history

**Comment by zackmuc** [comment]
I am testing the 1.7.0 beta, and it solved all problems i had. the only problem is that this beta will be automatically disabled all 2 weeks, when is the release planed?

**Comment by elvary** [comment]
> I am testing the 1.7.0 beta, and it solved all problems i had. the only problem is that this beta will be automatically disabled all 2 weeks, when is the release planed?

what are the problems you had and how long have u been testing 1.7.0 beta?

**Comment by zackmuc** [comment]
Testing it 2 months, i have no problems .. only that edge automatically removes beta plugins after 2 weeks automatically and so i always have to reinstall it.

**Comment by lionkmp** [comment]
Hi, When I try to install 1.7 beta, Firefox redirects me here: (I didn't try the extension yet, but I don't want to risk my bookmarks with the old 1.5.2, based on feedback you got on the Add-On site ;-) )

https://support.mozilla.org/en-US/kb/add-on-signing-in-firefox?as=u&utm_source=inproduct

**Comment by fastcat001** [comment]
It seems this amazing project has been abandoned.

**Comment by fastcat001** [comment]
> 

Do all chromium browsers have the same issue?

**Comment by fastcat001** [comment]
> Testing it 2 months, i have no problems .. only that edge automatically removes beta plugins after 2 weeks automatically and so i always have to reinstall it.

Do all chromium browsers have the same issue?

**Comment by paulwratt** [comment]
it appears the branch they is referring to has gone (been folded in to master?)

**Comment by nero120** [comment]
Yes it's been merged. New version is code complete and ready for release but it needs extensive testing as it's a complete code re-write and as such I just haven't had time.

TBH I'm considering just releasing it at this point. It's had a fair bit of beta testing by others (and myself) so we could just go with it and see what happens. Hopefully it solves more problems than it creates!

I'm crazy busy at work for the next week or so, so not a good time to release that now. Once my current project is finished I'll release it and we'll see what happens 🙏

**Comment by paulwratt** [comment]
yeah, just release it.

leave "stable version" as well (unless that conflicts with the server)

**Comment by zackmuc** [comment]
1.7.0 is working fine

**Comment by ShayBox** [comment]
Works great for me, fixes Vivaldi bookmarks not being delete-able, syncing into trash, and duplicating :+1:

**Comment by STPKITT** [comment]
Looking at the roadmap there I wondered would initial importing tags from Firefox technically be possible? It's the one thing which stops me from really using the otherwise awesome xBrowserSync since over the years I added so many tags to my Firefox bookmarks without which I would not be able to find the bookmarks I'm looking for.

**Comment by Rami-Ha** [comment]
Release it as stable. We're all waiting

**Comment by kkuhle** [comment]
@nero120 - Please release soon! :) I'm using 1.6 right now because 1.7 mobile app doesn't work for me, but 1.6 works fine. I haven't had any issues with this version other than it gets removed from chrome/edge automatically (it seems like every week) which is quite a pain.

**Comment by kkuhle** [comment]
> Testing it 2 months, i have no problems .. only that edge automatically removes beta plugins after 2 weeks automatically and so i always have to reinstall it.

@zackmuc  have you found any way to get it to stop removing? I use Chrome and Edge and it is frustrating that it gets removed. It feels like it's more like 5 days for me since it gets removed and with liking to use a different keyboard shortcut, it is tedious to re-add and update the shortcut so frequently.
